backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -914,15 +930,21 @@ public ProjectPaginationDTO findProjects(
                 ? (UserInfo) host.getCollaboratorMessage()
                 : currentLoginUserInfo;
         if (userInfo != null) {
-          accessibleProjectIds =
-              ModelDBUtils.filterWorkspaceOnlyAccessibleIds(
```

Comment:
I don't understand why we had to change the method.

Reply:
Internally `filterWorkspaceOnlyAccessibleIds`  method called the GetResourcesItem method and it is returning list of accessibleProjectIds so we can not build the GetResources Map using only ids so i have extracted that UAC endpoint here.